woocommerce-gateway-stripe.php

```diff
@@ -6,10 +6,10 @@
  * Author: WooCommerce
  * Author URI: https://woocommerce.com/
  * Version: 7.3.0
- * Requires at least: 5.9
+ * Requires at least: 6.0
  * Tested up to: 6.1.1
```

Comment:
Since we are running tests on [wordpress-6.2](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2610/files#diff-8b304272438502fbcb449090052d2e707da4b71c84982948503c8f5f6532b0adR26), should we bump this up to 6.2?

Reply:
Good catch, fixed in db8c358.